charts/sorry-cypress/templates/ingress-s3.tpl

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.s3.ingress.enabled -}}
+{{- $v1Networking := .Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress" -}}
+{{- $fullName := include "sorry-cypress-helm.fullname" . -}}
+{{- if $v1Networking }}
+apiVersion: networking.k8s.io/v1
+{{- else if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
+{{- end }}
+kind: Ingress
+metadata:
+  name: {{ $fullName }}-s3
+  labels:
+    {{- include "sorry-cypress-helm.labels" . | nindent 4 }}
+    {{- with .Values.s3.ingress.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end}}
+  annotations:
+    nginx.ingress.kubernetes.io/upstream-vhost: {{ include "s3Host" . }}
+    nginx.ingress.kubernetes.io/enable-cors: "true"
+    nginx.ingress.kubernetes.io/cors-allow-origin: "*"
+    {{- with .Values.s3.ingress.annotations }}
```

Comment:
What about add this annotation, `nginx.ingress.kubernetes.io/cors-allow-methods: "POST, GET, PUT, DELETE, HEAD"`, according to [documentation](https://sorry-cypress.dev/installation/aws#manually-setup-s3-bucket-for-screenshots-and-videos)?